src/Core/Tool/Authorizer/PostBulkAuthorizer.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Ushahidi Post Bulk Authorizer
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Tool\Authorizer;
+
+use Ushahidi\Core\Entity;
+use Ushahidi\Core\Tool\Authorizer;
+use Ushahidi\Core\Tool\Permissions\Acl;
+use Ushahidi\Core\Tool\Permissions\Permissionable;
+use Ushahidi\Core\Traits\AdminAccess;
+use Ushahidi\Core\Traits\UserContext;
+use Ushahidi\Core\Traits\ParentAccess;
+use Ushahidi\Core\Traits\PrivAccess;
+use Ushahidi\Core\Traits\PrivateDeployment;
+use Ushahidi\Core\Traits\PermissionAccess;
+use Ushahidi\Core\Traits\Permissions\ManageSettings;
+
+// The `FormAuthorizer` class is responsible for access checks on `Forms`
+class PostBulkAuthorizer implements Authorizer, Permissionable
```

Comment:
I think we could probably avoid the additional authorizer by treating `bulk_update` and `bulk_delete` as permissions similar to `update` and `delete` right now.